### PR TITLE
i18n(QB) translating strings

### PR DESCRIPTION
**Pull request by joebordes**

### CODE REVIEW

**Comment by joebordes** [comment]
Hi @kdoh ,

Here is another directory of changes.

I don't know how to translate these two blocks:

https://github.com/metabase/metabase/pull/6374/files#diff-6a19a8788c93235b7a1d182a660096feL53

https://github.com/metabase/metabase/pull/6374/files#diff-ce3ecc7c4b390728fd6a81b319ab37feL30

Please have a look in general and get back to me when you can.

File: frontend/src/metabase/qb/components/TimeseriesFilterWidget.jsx [hunk 1]
```diff
@@ -103,16 +103,16 @@ export default class TimeseriesFilterWidget extends Component {
                 currentFilter
             ).join(" - ");
             if (currentFilter[0] === ">") {
-                currentDescription = "After " + currentDescription;
+                currentDescription = t`After`+ " " + currentDescription;
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/TimeseriesFilterWidget.jsx]: I realize this was this way before you added the t tag but to make things a little cleaner I might convert these over to something like
```jsx
t`After ${currentDescription}`
```
just to keep things a bit more maintainable. Having to do `+ " "` to add an extra space could get a little fragile.

File: frontend/src/metabase/qb/components/actions/CommonMetricsAction.jsx [hunk 2]
```diff
@@ -16,9 +16,10 @@ export default ({ question }: ClickActionProps): ClickAction[] => {
     }
 
     const activeMetrics = query.table().metrics.filter(m => m.isActive());
+    const mname = <strong>{metric.name}</strong>
     return activeMetrics.slice(0, 5).map(metric => ({
         name: "common-metric",
-        title: <span>View <strong>{metric.name}</strong></span>,
+        title: <span>{t`View  ${ mname }`}</span>,
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/actions/CommonMetricsAction.jsx]: You can convert this to a variable but you can also use `jt` to handle jsx tags. It looks something like this

```jsx
<span>{ jt`View ${<span>{metric.name}</span>}`}</span>
```

[Here's an example](https://github.com/metabase/metabase/pull/6278/files#diff-9881a49f8bf82cd653f982fbb650c80bR181) from our alerts feature branch.

File: frontend/src/metabase/qb/components/actions/PivotByAction.jsx [hunk 3]
```diff
@@ -42,20 +42,14 @@ export default (name: string, icon: string, fieldFilter: FieldFilter) =>
         if (breakoutOptions.count === 0) {
             return [];
         }
-
+        const breakoutname = <span className="text-dark">{name.toLowerCase()}</span>
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/actions/PivotByAction.jsx]: Same thing here r.e. use of `jt` over creating a variable.

File: frontend/src/metabase/qb/components/actions/UnderlyingRecordsAction.jsx [hunk 4]
```diff
@@ -14,19 +14,13 @@ export default ({ question }: ClickActionProps): ClickAction[] => {
     if (!(query instanceof StructuredQuery) || query.isBareRows()) {
         return [];
     }
-
+    const dname = <span className="text-dark">{query.table().display_name}</span>
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/actions/UnderlyingRecordsAction.jsx]: Same here r.e. var.

File: frontend/src/metabase/qb/components/drill/QuickFilterDrill.jsx [hunk 5]
```diff
@@ -51,7 +51,7 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
                 section: "filter",
                 title: (
                     <span>
-                        View this
+                        {t`View this`}
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/drill/QuickFilterDrill.jsx]: I *think* you can do something like this to make this more contextual.

```jsx
<span>{ jt`View this ${singularize(stripId(column.display_name))}'s` }</span>
```

File: frontend/src/metabase/qb/components/drill/UnderlyingRecordsDrill.jsx [hunk 6]
```diff
@@ -27,7 +27,7 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
         {
             name: "underlying-records",
             section: "records",
-            title: "View " +
+            title: t`View` + " " +
```
Comment (kdoh) [inline frontend/src/metabase/qb/components/drill/UnderlyingRecordsDrill.jsx]: For this one I think the following should work. I suppose there's a chance for something like this that the broken up words could create some slightly unnatural sentences in certain languages if the proper context / tense isn't presented to the translator but I'm not sure of a way around this.

```jsx
title: t`View ${inflect(t`these`, count, t`this`, t`these`)} ${inflect(query.table().display_name, count}`
```

**Review by kdoh — COMMENTED** [review]
Looks pretty good! I tried to address a couple of your questions. Let me know if those solutions work. Thanks for all your help on help on getting Metabase ready to translate @joebordes!

**Comment by joebordes** [comment]
Ok, I think I got it. Thanks!!

Have a look and let me know if you need me to change anything else. I will fix the pulse changes and try to find some time to get at another directory.

**Review by kdoh — APPROVED** [review]
👍

**Review by kdoh — CHANGES_REQUESTED** [review]
Hey @joebordes. Just tried this locally and it looks like the webpack build doesn't actually build. You'll need to  import `jt` explicitly in the files that use it and I believe and there's also a lint error in one or two places. Mind running `yarn run lint` and `yarn run test` to make sure everything is working as intended?

**Comment by joebordes** [comment]
Hi @kdoh 
I passed lint and test and fixed all the errors except one which I do not know how to fix:

```
LINT
/space/reps/metabase/frontend/src/metabase/query_builder/containers/ArchiveQuestionModal.jsx
  3:19  error  Unable to resolve path to module 'c-3po'  import/no-unresolved

TEST
 FAIL  frontend/test/modes/drills/PivotByCategoryDrill.integ.spec.js
  ● Test suite failed to run

    Cannot find module 'c-3po' from 'AggregationPopover.jsx'
```

Let me know what you want me to do.

**Comment by kdoh** [comment]
Closing in favor of #6442

**Pull request closed without merging**